cmd/ethereum_rust_l2/src/commands/stack.rs

```diff
@@ -26,21 +26,31 @@ pub(crate) enum Command {
         l1: bool,
         #[clap(long, help = "Shuts down the L2 node.", default_value_t = true)]
         l2: bool,
+        #[clap(short = 'y', long, help = "Forces the restart without confirmation.")]
```

Comment:
I think you forgot to customize the message for each command (_Forces the ~~restart~~ shutdown_)